core/contracts/financial-templates/implementation/TokenFactory.sol

```diff
@@ -4,12 +4,17 @@ import "../../common/interfaces/ExpandedIERC20.sol";
 
 
 /**
- * @notice A factory for creating new mintable and burnable tokens.
+ * @title Factory for creating new mintable and burnable tokens.
  */
+
 contract TokenFactory {
     /**
-     * @notice Create a new token and return to the caller. The caller will become the only minter and burner
-     * and the new owner capable of adding new roles.
+     * @notice Create a new token and return to the caller.
+     * @dev The caller will become the only minter and burner and the new owner capable of adding new roles.
+     * @param tokenName used to describe the new token.
+     * @param tokenSymbol short ticker abbreviation of the name. Ideally < 5 chars.
```

Comment:
Is 5 characters the metamask limit?

Reply:
it's not a strict limit but it's the recommended number from the standard.